ci/integration-tests/src/tests/cw721_base_test.rs

```diff
@@ -0,0 +1,80 @@
+use crate::helpers::chain::Chain;
+use cosmwasm_std::Empty;
+use test_context::test_context;
+
+#[test_context(Chain)]
+#[test]
+#[ignore]
+fn execute_transfer_nft(chain: &mut Chain) {
+    let user_addr = chain.users["user1"].account.address.clone();
+    let user_key = chain.users["user1"].key.clone();
+    let receiver = chain.users["user2"].account.address.clone();
+
+    let msg = cw721_base::msg::InstantiateMsg {
+        name: "token".to_string(),
+        symbol: "nonfungible".to_string(),
+        minter: user_addr.clone(),
+    };
+
+    let res = chain
+        .orc
+        .instantiate(
+            "cw721_base",
+            "cw721_base_init",
+            &msg,
+            &user_key,
+            Some(user_addr.clone()),
+            vec![],
+        )
+        .unwrap();
+
+    chain
+        .orc
+        .contract_map
+        .add_address("cw721_base", res.address)
+        .unwrap();
```

Comment:
Nit: you can remove this part, as cosm_orc.instantiate() automatically does this for you after instantiating.

Reply:
nice!